domain/modeldefaults/service/service.go

```diff
@@ -41,6 +41,12 @@ type State interface {
 	// set cloud region.
 	ModelCloudRegionDefaults(context.Context, coremodel.UUID) (map[string]string, error)
 
+	// ModelMetadataDefaults is responsible for providing metadata defaults for a
+	// models config. These include things like the models name and uuid.
```

Comment:
apostrophe in a couple of palces

Reply:
I don't get why model name and uuid are considered defaults. These are invariant attributes of a model used at create time - one supplied by the user, one generated. This differs from defaults which are (db persisted) "out of the box" values to use for mutable config attributes if not supplied by the user. Mixing these very different concepts and obtaining them via this method call seems wrong. Perhaps it's just the method name that should be changed, eg "CoreModelAttributes" or something.

Reply:
As discussed we are introducing it in this manner as the values will eventually go. For the moment it is the safest and most elegant way to keep driving them back into the model config.